kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -346,6 +346,28 @@ class BinBasedFastPointLocator
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This operation is defined to the the corresponding container type
+     * @param rModelPart The model part to get the element container
+     * @return The corresponding element array
+     */
+    static inline PointerVectorSet<Element, IndexedObject>::ContainerType GetContainerType(ModelPart& rModelPart, PointerVectorSet<Element, IndexedObject>::ContainerType rContainerArray)
```

Comment:
function  is not getting the `Type` but the container ... is it not ? so in that case GetContainer is more suitable.